common/app/views/fragments/atoms/storyquestions.scala.html

```diff
@@ -16,56 +16,45 @@
-                                * show the form allowing a user to submit their email address and receive an answer.
-                                *********************@
+                <button type="submit" class="js-storyquestion-vote-button storyquestion-vote-button button button--primary">Vote</button>
 
-                                @if("test/test" == storyquestions.data.relatedStoryId) {
-                                    @storyquestions.atom.labels.find(_.length == 4).map { listId =>
-                                        <form id="js-storyquestion-email-signup-form-@question.questionId" class="is-hidden storyquestion-email-signup-form js-storyquestion-email-signup-form form" data-question-id="@question.questionId">
-                                            <div class="form-field js-storyquestion-email-signup-input-container">
-                                                <input class="text-input js-storyquestion-email-signup-input" type="email" name="email" placeholder="Email address" required />
-                                            </div>
-                                            <input class="" type="hidden" name="listId" value="@listId" />
-                                            <button type="submit" class="js-storyquestion-email-signup-button button button--primary button--with-input">@fragments.inlineSvg("envelope", "icon", Seq("submit-input__icon"))Sign up</button>
-                                        </form>
-                                    }
-                                }
-                            </div>
+                <button type="submit" class="is-hidden js-storyquestion-email-question-button storyquestion-email-question-button button button--primary">Email me</button>
+
+                @storyquestions.atom.labels.find(_.length == 4).map { listId =>
+                    <form action="#" class="is-hidden storyquestion-email-signup-form js-storyquestion-email-signup-form form">
+                        <div class="form-field js-storyquestion-email-signup-input-container">
+                            <input class="text-input js-storyquestion-email-signup-input" type="email" name="email" placeholder="Email address" required />
                         </div>
-                    }
-                </p>
+                        <input class="" type="hidden" name="listId" value="@listId" />
+                        <button type="submit" class="js-storyquestion-email-signup-button button button--primary button--with-input">
+                            @fragments.inlineSvg("envelope", "icon", Seq("submit-input__icon"))
+                            Sign up
+                        </button>
+                    </form>
+                }
+
+                <div class="is-hidden js-storyquestion-email-nospam">No spam, just one email</div>
```

Comment:
can you give an `id` to this too? and then add `aria-describedby="id-of-the-div"` on the input field above